### DISCUSSION TITLE
Column Relationship

### DISCUSSION THREAD

**Discussion by mentiantonio** [rappasoft/laravel-livewire-tables#257] [Q&A]
I have this on users model

public function addresses()
    {
        return $this->hasMany(Address::class);
    }

Inside my table component i have 

 public function query(): Builder
    {
        return User::with('addresses');
    }

but i don't see the related column on the view 

Column::make('Phone', 'addresses.phone')

What i am doing wrong?

**Comment by 22syamsudar** [comment]
i think u can just use like this..

			Column::make('Phone') 
				->format(function($row) {
					return $row['addresses']['phone'];
				})   
				->asHtml(), 

hope it answers yours...

**Comment by lrljoe** [comment]
This is present in v2+
